src/document/advanced.rs

```diff
@@ -25,6 +27,21 @@ pub trait DocumentAdvancedOps {
         data: &[u8],
         encrypt_opts: &DocumentEncryptOpts,
     ) -> Result<DocumentEncryptUnmanagedResult>;
+
+    /// (Advanced) Decrypt a document not managed by the ironcore service. Both the encrypted
+    /// data and the encrypted deks need to be provided.
+    ///
+    /// The webservice is still needed to transform a chosen encrypted dek so it can be decrypted
+    /// by the caller's private key.
+    ///
+    /// # Arguments
+    /// - `encrypted_data` - Encrypted document
+    /// - `encrypted_deks` - Associated encrypted DEKs for the `encrypted_data`
+    fn document_decrypt_unmanaged(
+        &self,
+        encrypted_data: &[u8],
```

Comment:
We have these in the opposite order in IronWeb. Can we reorder them?

Reply:
Slight push-back... If we think of unmanaged decrypt as a variant of normal decrypt it makes more sense to me to add the new parameter onto the end of the function call.

Reply:
Changed IronWeb.